new watchfaces: "kitt" and "arc"

jgibbon opened the pull request:
These are two new watchfaces:
- Kitt (looks nicer on square watches)
- Arc (looks nice on both round and square; has a bunch of variants)

@eLtMosen has tried them on lenok (thank you!), I've tested dory.

enjoy!

FlorentRevest commented on kitt/usr/share/asteroid-launcher/watchfaces/kitt.qml:
```diff
@@ -0,0 +1,230 @@
+    }
+
+
+    Connections {
```
This is not needed

jgibbon replied:
Oh, that's right, forgot to remove the localeManager Connection.
Thanks, I'll remove it!

FlorentRevest commented on arc/usr/share/asteroid-launcher/watchfaces/Arc.qml:
```diff
@@ -0,0 +1,370 @@
+            styleColor: watchFace.arccolor
+        }
+
+        ListView {
```
I have the feeling that this is an overly complex solution :P

jgibbon replied:
You feeling might be correct. 
My Goal was to simplify the variant usages (qml files using Arc { } ) and I might have gone too far.
If you think it's worth it, I'll review and refactor when time allows before opening another pull request.

FlorentRevest replied:
Don't worry, I can merge with that. It's just so that you know.
I don't have the time to think of something better either but I'm pretty sure there must be a simpler way.

FlorentRevest commented on arc/usr/share/asteroid-launcher/watchfaces/Arc-black-pie-noseconds.qml:
```diff
@@ -0,0 +1,11 @@
+Arc {
```
I don't see where all those Arc-*.qml items are being used, could you clarify?

jgibbon replied:
Those are variants that just look a bit different – black vs. white; arc vs. pie chart; seconds visible vs. not visible. They are listed on their own in the watchfaces settings.

FlorentRevest replied:
Fair enough, I didn't understand directly. This can be merged as well

FlorentRevest commented:
Hey jgibbon!
Thanks a lot for your work :) I just had a few comments. Otherwise, I'll get this PR merged as soon as that will be fixed

jgibbon commented:
Hi FlorentRevest,
thanks for reviewing!
As I wrote in the replies, I'll be happy to rework the model situation and remove the connections if you think they look nice enough. Might take a while, though. ;)

cheers

jgibbon commented:
Ok, thanks!
If I find the time and think of something clever, I can revisit the model thing later on, even if this version is merged.

I've removed the "Connections" (but due to lack of time and my usb dock, I just removed it here via web interface without testing!) from "kitt".

FlorentRevest commented:
Thank you !
